### PR TITLE
Release 0.4.1

### PR DESCRIPTION
**Pull request by surajssd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+#### Features
+
+- feat: add severity labels to MetalLB alerts ([#925](https://github.com/kinvolk/lokomotive/pull/925)).
+
+#### Bug fixes
+
+- Override memory limits of rook operator to 512Mi ([#938](https://github.com/kinvolk/lokomotive/pull/938)).
+- Fix envoy grafana dashboard errors ([#969](https://github.com/kinvolk/lokomotive/pull/969)).
+- MetalLB: Fix regressions of tolerations and nodeSelectors ([#927](https://github.com/kinvolk/lokomotive/pull/927)).
+- Fix controlplane components order ([#937](https://github.com/kinvolk/lokomotive/pull/937)).
```
Comment (invidian) [inline CHANGELOG.md:16]:
```suggestion
- Fix controlplane components update order ([#937](https://github.com/kinvolk/lokomotive/pull/937)).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,34 @@
...
+
+#### Docs
+
+- docs: fix etcd version upgrade sed expression ([#921](https://github.com/kinvolk/lokomotive/pull/921)).
+- docs: fix rook version update command ([#930](https://github.com/kinvolk/lokomotive/pull/930)).
+
+#### Development
+
+- Fix output of `convertNodeSelector` in rook ([#945](https://github.com/kinvolk/lokomotive/pull/945)).
+- Dex convert to helm chart and update to v2.25.0 ([#962](https://github.com/kinvolk/lokomotive/pull/962)).
```
Comment (invidian) [inline CHANGELOG.md:27]: It actually updates the component, maybe we should put it in updates section instead?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,34 @@
...
+- httpbin convert to helm chart ([#965](https://github.com/kinvolk/lokomotive/pull/965)).
+- FLUO: convert to Helm chart ([#935](https://github.com/kinvolk/lokomotive/pull/935)).
+- Makefile: Don't build before linting ([#901](https://github.com/kinvolk/lokomotive/pull/901)).
```
Comment (invidian) [inline CHANGELOG.md:30]: the PR actually adds a separate target to run linter without building.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,34 @@
...
+
+This is a patch release that mainly includes bug fixes.
+
+### Changes in v0.4.1
+
+#### Features
+
+- feat: add severity labels to MetalLB alerts ([#925](https://github.com/kinvolk/lokomotive/pull/925)).
+
+#### Bug fixes
```
Comment (invidian) [inline CHANGELOG.md:11]: Should we mention that we increased node ready and cluster ping timeouts?

**Review by invidian — CHANGES_REQUESTED** [review]
Nice work @surajssd. I'm thinking now, perhaps we should be going trough the list of the commits between the releases to find the highlights of the release, as PR titles sometimes do not get updated and also may contain many important changes.

File: CHANGELOG.md (hunk 3 above)
Comment (surajssd) [reply CHANGELOG.md:30]: But this is still development and not for the users, but for the devs. I have added a line about new target. But this should still remain in this section.

**Review by iaguis — DISMISSED** [review]
lgtm

**Review by rata — APPROVED** [review]
LGTM! :)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,38 @@
+## v0.4.1 - 2020-09-15
+
+This is a patch release that mainly includes bug fixes.
```
Comment (johananl) [inline CHANGELOG.md:3]:
```suggestion
This is a patch release which includes mainly bug fixes.
```

**Review by johananl — APPROVED** [review]
Suggested an improvement to the description. LGTM otherwise.

**Comment by johananl** [comment]
I assume we've tested upgrading from `v0.4.0`?

**Review by knrt10 — APPROVED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (invidian) [reply CHANGELOG.md]: Yes, it's fine. Just the title was wrong.

**Review by invidian — APPROVED** [review]
LGTM

**Review by invidian — APPROVED** [review]

**Review by knrt10 — APPROVED** [review]

**Pull request merged**